docs/ref/contrib/admin/index.txt

```diff
@@ -1576,9 +1579,9 @@ templates used by the :class:`ModelAdmin` views:
 
     .. admonition:: Note
 
-        Any ``choices`` attribute set on the formfield will limited to the form
-        field only. If the corresponding field on the model has choices set,
-        the choices provided to the form must be a valid subset of those
+        Any ``choices`` attributes set on the formfield will be limited to the 
```

Comment:
Not sure this is correct --- a formfield can only have a single "choices" attribute.